syntax = "proto3";
import "google/protobuf/any.proto";

message Apply_data
{
    uint32 apply_data_id = 1;
    uint64 apply_time =2;
}

message Apply_list
{
    uint32 apply_list_id = 1;
}

message Test
{
    int32 id = 1;
    google.protobuf.Any data = 2;
}
